util/src/builder/command.rs

```diff
@@ -604,13 +712,16 @@ impl NumberBuilder {
     /// Defaults to no choices.
     ///
     /// [`choice_localizations`]: Self::choice_localizations
-    pub fn choices(mut self, choices: impl IntoIterator<Item = (String, Number)>) -> Self {
+    pub fn choices<S: Into<String>, N: Into<Number>>(
```

Comment:
I don't believe we actually have `From<f64> for Number` implemented, so as-is this isn't too useful

Reply:
I replaced it with f64 like it's done in its other methods